pyaedt/application/Design.py

```diff
@@ -1025,103 +1049,90 @@ def oproject(self, proj_name=None):
 
     @property
     def oanalysis_setup(self):
-        """Property
-
-        :return: AnalysisSetup Module object
-
-        Parameters
-        ----------
+        """AnalysisSetup module object.
 
         Returns
         -------
+        type
+            AnalysisSetup module object.
 
         """
         return self.odesign.GetModule("AnalysisSetup")
 
     @property
     def odesktop(self):
-        """Property
-
-        :return: oDesktop Module object
-
-        Parameters
-        ----------
+        """Desktop module object.
 
         Returns
         -------
+        type
+            Desktop module object.
 
         """
         return self._desktop
 
     @property
     def desktop_install_dir(self):
-        """Property
-
-        :return: AEDT Install Dir
-
-        Parameters
-        ----------
+        """Desktop installation directory.
 
         Returns
         -------
+        str
+            AEDT installation directory.
+            
 
         """
         return self._desktop_install_dir
 
     @property
     def messenger(self):
-        """Property
-
-        :return: Messenger object that can be used for logging on log file and on AEDT Message Windows
-
-        Parameters
-        ----------
+        """Messenger object that is used to write logs to the log file and to AEDT Message windows.
 
         Returns
         -------
+        type
+            Messenger object.
 
         """
         return self._messenger
 
     @property
     def variable_manager(self):
-        """Property
-
-        :return: Variable maanager that can be used to create and manage Project, Design and PostProcessing Variables
-
-        Parameters
-        ----------
+        """Variable manager object that is used to create and manage project, 
+        design and postprocessing variables.
```

Comment:
```suggestion
        """Variable manager
        
         This object is used to create and manage project design and postprocessing variables.
```

Let's keep this short and sweet for the eventual table docs.